bosh-release/README.md

```diff
@@ -1,108 +1,253 @@
+        mysql -h $mysql_analytics_host -u $mysql_analytics_username -p$mysql_analytics_password -e "DROP DATABASE IF EXISTS "$event_store_db"; DROP DATABASE IF EXISTS "$processed_data_db"; DROP DATABASE IF EXISTS "$stats_db"; CREATE DATABASE "$event_store_db"; CREATE DATABASE "$processed_data_db"; CREATE DATABASE "$stats_db";"
 
-7. Add above distributions as blobs:
+        echo -e "\e[32m>> Creating tables... \e[0m"
+        mysql -h $mysql_apim_host -u $mysql_apim_username -p$mysql_apim_password -e "USE "$am_db"; SOURCE wso2am-2.1.0/dbscripts/apimgt/mysql5.7.sql; USE "$um_db"; SOURCE wso2am-2.1.0/dbscripts/mysql5.7.sql; USE "$reg_db"; SOURCE wso2am-2.1.0/dbscripts/mysql5.7.sql;"
 
-    ```bash
-    bosh -e vbox add-blob jdk-8u144-linux-x64.tar.gz oraclejdk/jdk-8u144-linux-x64.tar.gz
-    bosh -e vbox add-blob wso2am-2.1.0.zip wso2apim/wso2am-2.1.0.zip
-    bosh -e vbox -n upload-blobs
-    ```
 
-8. Create the WSO2 API Manager bosh release:
+6. Clone [bosh-deployment][3] git repo.
 
-    ```bash
-    bosh -e vbox create-release --force
-    ```
+        if [ ! -d bosh-deployment ]; then
+            echo -e "\e[32m>> Cloning https://github.com/cloudfoundry/bosh-deployment... \e[0m"
+            git clone https://github.com/cloudfoundry/bosh-deployment bosh-deployment
+        fi
 
-9. Upload the WSO2 API Manager bosh release to BOSH Director:
+7. Create bosh environment.
 
-    ```bash
-    bosh -e vbox upload-release
-    ```
+        if [ ! -d vbox ]; then
+            echo -e "\e[32m>> Creating envionment dir... \e[0m"
+            mkdir vbox
+        fi
+        
+        echo -e "\e[32m>> Creating environment... \e[0m"
+        bosh create-env bosh-deployment/bosh.yml \
+            --state vbox/state.json \
+            -o bosh-deployment/virtualbox/cpi.yml \
+            -o bosh-deployment/virtualbox/outbound-network.yml \
+            -o bosh-deployment/bosh-lite.yml \
+            -o bosh-deployment/bosh-lite-runc.yml \
+            -o bosh-deployment/jumpbox-user.yml \
+            --vars-store vbox/creds.yml \
+            -v director_name="Bosh Lite Director" \
+            -v internal_ip=192.168.50.6 \
+            -v internal_gw=192.168.50.1 \
+            -v internal_cidr=192.168.50.0/24 \
+            -v outbound_network_name=NatNetwork
 
-10. Download latest bosh-lite warden stemcell from bosh.io and upload it to BOSH Director:
-    
-    ```bash
-    wget https://s3.amazonaws.com/bosh-core-stemcells/warden/bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz
-    bosh -e vbox upload-stemcell bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz
-    ```
+8. Once VM with BOSH Director is running, point Bosh CLI to it, saving the environment with the alias `vbox`.
```

Comment:
I think better to write the word bosh either in uppercase or lowercase; BOSH

Reply:
Fixed in https://github.com/wso2/pivotal-cf-apim/pull/2